bbot/modules/baddns_direct.py

```diff
@@ -74,19 +74,19 @@ async def filter_event(self, event):
         if event.type == "STORAGE_BUCKET":
             if str(event.module).startswith("bucket_"):
                 return False
-            self.debug(f"Processing STORAGE_BUCKET for {event.host}")
+            self.debug(f"Processing STORAGE_BUCKET for {str(event.host)}")
```

Comment:
it's not necessary to use str() inside format strings since they're converted automatically.

Reply:
TIL